admin/class-monk-admin.php

```diff
@@ -1487,6 +1504,29 @@ public function medias_modal_filter( $query ) {
 	}
 
 	/**
+	 * Adds a language filter to the terms pages
+	 *
+	 * There are no hooks to use here, so we create the
+	 * components using the admin_footer action and move them to the right location
+	 *
+	 * @since    0.3.0
+	 *
+	 * @return void
+	 */
+	public function monk_add_term_language_filter() {
```

Comment:
Change from `monk_add_term_language_filter` to `monk_add_term_language_selector` to be more concise.